docs/my-website/docs/proxy/guardrails/acuvity_ai.md

```diff
@@ -0,0 +1,194 @@
+import Image from '@theme/IdealImage';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Acuvity Guardrails for LiteLLM
+
+Use **Acuvity** to detect **PII**, **Prompt Injection**, and other security risks in requests and responses.
+
+## 1. Installation
+
+Since Acuvity is an optional dependency, install it using Poetry:
+
+```shell
+poetry install --extras "acuvity"
+```
+
+Alternatively, if you are using `pip`:
+
+```shell
+pip install acuvity
+```
+
+## 2. Setup Guardrails on Acuvity
+
+### On more details on the usage of Acuvity with guardrails, please visit the https://docs.acuvity.ai/
+
+### **Define Guardrails for Different Stages**
+
+Acuvity supports guardrails at different stages of an LLM request:
+
+1. **Pre LLM API Call** - Validate and redact sensitive information before sending the request.
+2. **During LLM API Call** - Monitor for prompt injection or malicious content.
+3. **Post LLM API Call** - Analyze and filter responses for security violations.
```

Comment:
Rewrite these sections here a bit and be very specific about what they do:

1. pre call: **redact only** if applicable! Non-redaction guards will cause a failure. No policy decision is being made. Requests will always be allowed.
2. during call: applies policy decision on all requests. If they match the guard config, then requests will be rejected. **No redaction** is going to be applied, and guards that have redaction enabled will cause a failure.
3. post call: Can both redact and apply policy decision based on the provided guard config.